tests/gadget-zero/usb-gadget0.c

```diff
@@ -279,6 +311,7 @@ static void gadget0_set_config(usbd_device *usbd_dev, uint16_t wValue)
 	ER_DPRINTF("set cfg %d\n", wValue);
 	switch (wValue) {
 	case GZ_CFG_SOURCESINK:
+		test_unaligned = 0;
```

Comment:
Certainly, I wasn't sure if I could do that or if the state structure was reserved for the linux-gadget-compatible functionality.